cluster-sync/sync.sh

```diff
@@ -14,8 +14,8 @@ source ./cluster-sync/${KUBEVIRT_PROVIDER}/provider.sh
 CDI_INSTALL="install-operator"
 CDI_NAMESPACE=${CDI_NAMESPACE:-cdi}
 CDI_INSTALL_TIMEOUT=${CDI_INSTALL_TIMEOUT:-120}
-CDI_AVAILABLE_TIMEOUT=${CDI_AVAILABLE_TIMEOUT:-480}
-CDI_PODS_UPDATE_TIMEOUT=${CDI_PODS_UPDATE_TIMEOUT:-480}
+CDI_AVAILABLE_TIMEOUT=${CDI_AVAILABLE_TIMEOUT:-4800}
```

Comment:
can't it be a bit annoying to timeout only after 80 mins?

Reply:
ah, I can probably lower it back - I was so confused as to why it was failing on the CI. It was a completely different issue, but I wanted to be sure it's not some too-low timeout.